src/Components/Views/Balance/BalanceContainer.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import * as Types from '../../../types/Types';
+import {Skeleton} from "antd";
+import {BalanceView} from "./BalanceView";
+
+interface BalanceContainerProps {
+    userId: number;
+}
+
+interface BalanceContainerState {
+    balance: Types.Balance | null;
```

Comment:
You should prefer `undefined` over `null` in typescript https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined